service/worker/handler/migration_bucket_list_obj_handler.go

```diff
@@ -79,12 +80,12 @@ func (s *svc) HandleMigrationBucketListObj(ctx context.Context, t *asynq.Task) e
 		return err
 	}
 
-	// versioningConfig, err := fromClient.S3().GetBucketVersioning(ctx, p.Bucket)
-	// if err != nil {
-	// 	return fmt.Errorf("unable to get bucket versioning config: %w", err)
-	// }
+	versioningConfig, err := fromClient.S3().GetBucketVersioning(ctx, p.Bucket)
```

Comment:
is it makes sense to check state once, before creating a task and add `"versioned": true` to task payload instead?

aws docs says that versioning cannot be disabled, only suspended.

> You enable and suspend versioning at the bucket level. After you version-enable a bucket, it can never return to an unversioned state. But you can suspend versioning on that bucket.

https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html#versioning-states